src/data/events.ts

```diff
@@ -0,0 +1,83 @@
+import type { Image } from '@/components/ImageCarousel';
+
+type Month =
+    | 'JAN'
+    | 'FEB'
+    | 'MAR'
+    | 'APR'
+    | 'MAY'
+    | 'JUN'
+    | 'JUL'
+    | 'AUG'
+    | 'SEP'
+    | 'OCT'
+    | 'NOV'
+    | 'DEC';
+type Time = `${number}:${number}${'am' | 'pm'}`;
```

Comment:
I think we should use 24-hour time, thoughts?

Reply:
`time: string` ? XD